dev/bots/test.dart

```diff
@@ -274,6 +274,16 @@ Future<void> _runCommandsToolTests() async {
   );
 }
 
+Future<void> _runWebToolTests() async {
```

Comment:
Make it look like https://github.com/flutter/flutter/blob/4851bd51f9028a197f25649160a39d3d3d689945/dev/bots/test.dart#L256-L264